apps/server/src/modules/oauth/error/user-not-found-in-unprovisioned-school.loggable-exception.ts

```diff
@@ -0,0 +1,19 @@
+import { ErrorLogMessage, Loggable, LogMessage, ValidationErrorLogMessage } from '@src/core/logger';
+import { OAuthSSOError } from './oauth-sso.error';
+
+export class UserNotFoundInUnprovisionedSchoolLoggableException extends OAuthSSOError implements Loggable {
+	constructor(private readonly externalUserId: string, private readonly systemId: string) {
+		super('Provisioning of user with externalId: externalUserId failed', 'sso_user_notfound_in_unprovisioned_school');
```

Comment:
in_unprovisioned_school does not sound right. The school can be provisioned, but the school does not have provisioning of new users enabled. So something like 'user_not_found_after_provisioning', 'cannot_provision_user', 'user_provisioning_not_possible' or 'user_provisioning_disabled' would be right.